datepicker/views/index.html

```diff
@@ -0,0 +1,110 @@
+<index: attributes="class inline">
+{{if show}}
```

Comment:
Basically, one can view an if statement as a block - similar to an element - when it comes to indentation for views.